packages/strapi-plugin-email/admin/src/translations/dk.json

```diff
@@ -0,0 +1,4 @@
+{
+    "plugin.description.long": "Send e-mails.",
```

Comment:
Can you also translate this file?

Reply:
The file is already translated. It’s just the same in danish as in english